todo_app/data/trello_items.py

```diff
@@ -22,7 +22,7 @@ def get_items():
 
     query ={
         'cards' : 'open',
-        'card_fields' : 'name,idList',
```

Comment:
(not important, but worth noting that `listName` is a field that _we're_ adding, not one that Trello will actually use, so I suspect this change is being ignored)